### ISSUE TITLE
How to start replay of a child process skipping the earlier processing?

### ISSUE THREAD

**Issue by githubsidr** [rr-debugger/rr#2744]
Hi,

    I have an executable that forks/execs multiple programs *serially*, and once all the child processes are done, the parent exits. I can see the child processes via "rr ps". However, when I replay one of the child processes using "rr replay -p <pid>", I see the prints and logs of all the **earlier** child processes. Is this expected? This slows down the start to debug the child process of my interest. Is there any switch or workaround to jump to the start of the child process instantly?

Regards,
Siddharth

**Comment by rocallahan** [comment]
> Is this expected?

Yes. `rr replay -q` will silence those echoed writes. Honestly I'd be surprised if just echoing those to a terminal is a performance issue.

One thing you could do if you have to is do `rr replay -M -s 1999 -p <pid>|grep <pid>` in one terminal and then run gdb like this:
```
gdb '-l' '10000' '-ex' 'set sysroot /' '-ex' 'target extended-remote 127.0.0.1:1999' <exe>
```

**Comment by githubsidr** [comment]
Thanks. It's not just the echoing prints on the terminal. It's that the earlier child processes consume a lot of CPU and can have been running for hours. So the replay of a subsequent process can be stalled for a long time.

**Comment by rocallahan** [comment]
Ah. That's a fundamental design issue of rr at the moment and there is no way to work around it.
